packages/features/tasker/tasks/scanWorkflowBody.ts

```diff
@@ -90,30 +100,29 @@ export async function scanWorkflowBody(payload: string) {
 
     const isSpam = await iffyScanBody(workflowStep.reminderBody, workflowStep.id);
 
-    if (isSpam) {
-      if (workflowStep.workflow.user?.whitelistWorkflows) {
-        log.warn(
-          `For whitelisted user, workflow step ${workflowStep.id} is spam with body ${workflowStep.reminderBody}`
-        );
-        return;
-      }
-
+    if (isSpam && !workflowStep.workflow.user?.whitelistWorkflows) {
```

Comment:
whitelistWorkflows should be set as verified and we want to schedule reminders cc @joeauyeung

Reply:
That made the unit tests fail. Am I wrong here? I assumed that if whitelistWorkflows is true, we never want to mark it as spam

Reply:
aside currently there is a warning if a white listed user has been maraked as spam, this removes that message - but if we don't want that message we can check if the user is whitelisted and not trigger iffy at all

Reply:
aside warning message is still here: https://github.com/calcom/cal.com/pull/21201/files/e66134b0b41a6ae623436b57845129596bbfee76#diff-06234ba1496110f9113e037dc08e17381b14ed44e74b8ab59585a57783626ed0R113

Reply:
aside you're right! but it misses the isSpam && check, so that warning message will log for all workflows